cypress-tests/readme.md

```diff
@@ -1,45 +1,196 @@
 # Cypress Tests
 
 ## Overview
+
 This Tool is a solution designed to automate testing for the [Hyperswitch](https://github.com/juspay/hyperswitch/) using Cypress, an open-source tool capable of conducting API call tests and UI tests. This README provides guidance on installing Cypress and its dependencies.
 
 ## Installation
 
 ### Prerequisites
+
 Before installing Cypress, ensure you have the following prerequisites installed:
+
 - npm (Node Package Manager)
 - Node.js (18.x and above)
 
 ### Run Test Cases on your local
+
 To run test cases, follow these steps:
 
 1. Install Cypress
+
    ```shell
    npm install cypress --save-dev
    ```
-3. Clone the repository and switch to the project directory:
+
+2. Clone the repository and switch to the project directory:
 
    ```shell
    git clone https://github.com/juspay/hyperswitch
    cd cypress-tests
    ```
-4. Set environment variables for cypress
+
+3. Set environment variables for cypress
+
    ```shell
    export CYPRESS_CONNECTOR="connector_id"
    export CYPRESS_BASEURL="base_url"
    export DEBUG=cypress:cli
    export CYPRESS_ADMINAPIKEY="admin_api_key"
    ```
 
-5. Run Cypress test cases
-    To run the tests in a browser in interactive mode run the following command
-    ```
-    npm run cypress 
-    ```
-    To run the tests in headless mode run the following command
-    ```
-    npm run cypress:ci
-    ```
+4. Run Cypress test cases
+
+   To execute a connector create test, ensure the connector details are included in the `creds.json` file. Then, integrate the path in the `command.js` as follows:
```

Comment:
`command.js` should be `commands.js`?
Here and elsewhere in this doc.

Reply:
Sure ill add it in other PR